src/src/Helpers/Networking.cpp

```diff
@@ -1544,7 +1544,26 @@ int http_authenticate(const String& logIdentifier,
     event += '=';
     event += httpCode;
     eventQueue.addMove(std::move(event));
+    #if FEATURE_THINGSPEAK_EVENT
+      // Generate event with the response of a 
+      // one value one channel tingspeak request
+      // e.g. (sendToHTTP,api.thingspeak.com,80,/channels/143789/fields/5/last)
+      // where first eventvalue is the channel number, the second the field number
+      // and the third is the value received by the request
+      if (equals(host, F("api.thingspeak.com")) && equals(uri.substring(uri.length() - 5, uri.length()), F("/last")) && httpCode == 200) {
+      String revent = F("reply#");
```

Comment:
Since the event is very specific to thingspeak, shouldn't the event be less generic?
Like `ThingspeakReply#`

Reply:
i will go with reply#thingspeak since the url is always the same anyway...

Reply:
Still the eventvalues are very specific to thingspeak...
And you can also match such events like this:
```
on reply do

endon
```

Reply:
ok...so i would only use `ThingspeakReply#` since adding the host is kind of unneccesary?

Like:
```
String revent = F("ThingspeakReply#");
        //revent += host;
        revent += '=';
        ...
```

Reply:
> Still the eventvalues are very specific to thingspeak... And you can also match such events like this:
> 
> ```
> on reply do
> 
> endon
> ```

you mean 
 ```
 on reply* do
 
 endon
```

Reply:
If you don't add something betweek `#` and `=`, then it should be left empty I guess.

On taskvalue events, like bme#temp=12.3 you can 'catch' the event by using `on bme do`... At least that was my intent.
Not sure if that is somewhere mentioned in the docs explicitly though.

If you try matching with wildcards, you may get strangs results as `replyfoo` or `replyfoo#` may match but also  `reply#foo=` or  `reply=`.

Reply:
just checked.. on taskvalue events `on bme do` and `on bme# do` does not work...